doc/tutorial/machine_learning_map/index.rst

```diff
@@ -11,9 +11,9 @@ data and different problems.
 
 The flowchart below is designed to give users a bit of a rough guide on how to approach
 problems with regard to which estimators to try on your data. Click on any estimator in
-the chart below to see its documentation. Use scroll wheel to zoom in and out, and click
-and drag to pan around. You can also download the chart:
-:download:`ml_map.svg <../../images/ml_map.svg>`.
+the chart below to see its documentation. The 😭 emoji stands for "not working". Use
+scroll wheel to zoom in and out, and click and drag to pan around. You can also download
```

Comment:
Honest question about the emoji, would it show as hoover text? What is the thing that doesn't work, the link to the estimator? Or in which scenario one would expect the emoji to appear? Is it a draw.io standard thing?

Reply:
The emoji is a short replacement for "not working" as is used in the original graph, on the orange arrows. It means that if this estimator does not work, then try xxx. Should I clarify somehow or simply remove this sentence?

Original:

![image](https://github.com/scikit-learn/scikit-learn/assets/108576690/e5a214b7-9966-40d9-9274-4704befd7800)

New:

![image](https://github.com/scikit-learn/scikit-learn/assets/108576690/e3db6e6c-b77e-4abd-9891-22135da7e0fb)

Reply:
Ah sorry! I didn't notice it in the new image. Then I think we can use a more expressive sentence ("not working" was probably minimalist for the sake of fitting the map). In the text we could say something more similar to 
> The 😭 emoji is to be read as "if this estimator does not achieve the desired outcome, then try xxx"